cprover_bindings/src/goto_program/symbol.rs

```diff
@@ -20,8 +20,6 @@ pub struct Symbol {
     /// Contracts to be enforced (only supported for functions)
     pub contract: Option<Box<FunctionContract>>,
 
-    /// Optional debugging information
```

Comment:
Why are we deleting this?

Reply:
I don't know why we had it in the first place: none of the fields below this are "debug" information, they are rather essential to describe that symbol.